src/main/java/org/killbill/billing/plugin/adyen/api/mapping/UserDataMappingService.java

```diff
@@ -69,8 +73,79 @@ public static UserData toUserData(@Nullable final Account account, final Iterabl
         // set ip
         userData.setShopperIP(PluginProperties.findPluginPropertyValue(AdyenPaymentPluginApi.PROPERTY_IP, properties));
 
-        return userData;
-    }
+        //determine AccountAge Indicator
+        final String accountAgeIndicator = PluginProperties.findPluginPropertyValue(ACCOUNT_AGE_INDICATOR,properties);
+        final Optional<String> optionalAccountAgeIndicator = toLastName(accountAgeIndicator,account);
+        final String accAgeIndicator = optionalAccountAgeIndicator.isPresent() ? optionalAccountAgeIndicator.get() : null;
+        userData.setAccountAgeIndicator(accAgeIndicator);
+
+        final String accountChangeDate = PluginProperties.findPluginPropertyValue(ACCOUNT_CHANGE_DATE,properties);
+        if(accountChangeDate != null) {
+            userData.setAccountChangeDate(DateTime.parse(accountChangeDate));
+        }
+        final String accountChangeIndicator = PluginProperties.findPluginPropertyValue(ACCOUNT_CHANGE_INDICATOR,properties);
+        userData.setAccountChangeIndicator(accountChangeIndicator);
+
+        final String accountCreationDate = PluginProperties.findPluginPropertyValue(ACCOUNT_CREATION_DATE,properties);
+        if(accountCreationDate != null) {
+                userData.setAccountCreationDate(DateTime.parse(accountCreationDate));
+        }
+
+        final String passwordChangeDate = PluginProperties.findPluginPropertyValue(PASSWORD_CHANGE_DATE,properties);
+        if(passwordChangeDate != null) {
+            userData.setPasswordChangeDate(DateTime.parse(passwordChangeDate));
+        }
+        final String passwordChangeDateIndicator = PluginProperties.findPluginPropertyValue(PASSWORD_CHANGE_DATE_INDICATOR,properties);
```

Comment:
We don't need two properties for this. The change indicator can be derived from the change date property.